tests/src/Search/Web/TypeTest.php

```diff
@@ -25,7 +25,7 @@ public function typeProvider() : Traversable
             'application/vnd.elife.search+json' => 200,
             'application/vnd.elife.search+json; version=0' => 406,
             'application/vnd.elife.search+json; version=1' => 200,
-            'application/vnd.elife.search+json; version=2' => 406,
+            'application/vnd.elife.search+json; version=2' => 200,
```

Comment:
We should add a test for version 3 to return 406.

Reply:
@nlisgo added.